src/extension/tab/tab.ts

```diff
@@ -10,6 +10,13 @@ if (typeof document === "object" && document instanceof HTMLDocument) {
   const script = document.createElement("script");
   script.setAttribute("type", "module");
   script.setAttribute("src", chrome.extension.getURL("hook.js"));
-  const head = document.head || document.getElementsByTagName("head")[0] || document.documentElement;
-  head.insertBefore(script, head.lastChild);
+  document.addEventListener("DOMContentLoaded", () => {
+    const importMap = document.querySelector("script[type=\"importmap\"]");
+    if (importMap != null) {
+      importMap.parentNode?.insertBefore(script, importMap.nextSibling);
+    } else {
+      const head = document.head || document.getElementsByTagName("head")[0] || document.documentElement;
+      head.insertBefore(script, head.lastChild);
```

Comment:
This branching shouldn't actually even be necessary, since a dynamic load at DOM Content Loaded time would be deemed to always load after the import map so wouldn't interfere with the import map.

If wanting to load sooner, another option could be to check if there is already a module script, and if so trust that it is safe to do an injection, otherwise waiting for DOM content loaded.